### DISCUSSION TITLE
Micro timeout

### DISCUSSION THREAD

**Discussion by rolisz** [deta/cloud-docs#360] [Help]
I'm trying to generate some PDFs using PyPDF2. Locally, it runs in 2 seconds. On Deta Micros, it always times out (so it takes longer than 10 seconds). Any ideas why this huge discrepancy? 

I'm not trying to process a very complicated document, it's a 1 page PDF with a form in it, that is like 403 kb, so there's no reason for the processing to take 10 seconds.

**Comment by abdelhai** [comment]
gi, this could be due to the limited default memory. please request a ram increase https://form.deta.dev/memory
